src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs

```diff
@@ -200,9 +204,9 @@ private void PopulateAssemblies()
         /// This is not meant to be a general mechanism for injecting files that can't be handled
         /// through the normal NuGet package mechanisms.
         /// </summary>
-        private void PopulatePreResolvedNetStandardAssemblies()
+        private void PopulateExistingReferenceItems()
         {
-            var preResolvedAssemblyDependencies = new List<ITaskItem>();
+            var existingReferenceItemDependencies = new List<ITaskItem>();
             foreach (var reference in References)
             {
                 var packageName = reference.GetMetadata("NuGetPackageId");
```

Comment:
@ericstj I don't entirely follow. The `Type` metadata on what item? The `Reference`?